### DISCUSSION TITLE
[Resources] Looking for some getting started materials for code contributors

### DISCUSSION THREAD

**Discussion by akbatra567** [docsifyjs/docsify#1950] [General]
Might be a repeat question, but I couldn't find it so opening a discussions.
Do point me to existing links/documents if already exists.

Hello People,
I'm looking to start contributing to Docsify and just started to go through the code. Wanted to check if we already have a developer getting started document to understand which module, code performs what tasks, as well as understanding the user, actors and the business contracts for Docsify.

**Comment by sy-records** [comment]
Thank you for your thoughts.

But there is no manual for developers.

You can contribute from the code or the documentation and start by solving issues.
